### PR TITLE
Adding library, example and docs

### PR DESCRIPTION
**Pull request by BlitzCityDIY**
Adding library, example and docs for the FT5336 touch driver

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -23,17 +23,19 @@
...
-# TODO: Please Read!
-# Uncomment the below if you use native CircuitPython modules such as
-# digitalio, micropython and busio. List the modules you use. Without it, the
-# autodoc module docs will fail to generate with a warning.
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = [
+    "micropython",
+    "busio",
+    "adafruit_bus_device",
+    "adafruit_register",
```
Comment (makermelissa) [inline docs/conf.py:30]: I don't think these 2 lines are necessary here if they are libraries. Especially since you already have them under requirements.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Looks pretty good. Just one small change request. Also, nice use of the register library. I may have to update the CSTXXX library in the same way.

File: docs/conf.py (hunk 1 above)
Comment (makermelissa) [reply docs/conf.py:30]: Hmm, the error is odd and that adding it there fixed it. Neither of the 2 previous touch libs needed an autodoc_mock_imports and it's really only used for adding internal modules that haven't been added to Blinka.
- https://github.com/adafruit/Adafruit_CircuitPython_CST8XX/blob/main/docs/conf.py
- https://github.com/adafruit/Adafruit_CircuitPython_FocalTouch/blob/main/docs/conf.py

I'll see if I can find more info on the error.

Comment (makermelissa) [reply docs/conf.py]: Can you try removing all of the autodoc_mock_imports? Really, none of them should be needed.

Comment (BlitzCityDIY) [reply docs/conf.py]: ah, that did it. thank you!

Comment (makermelissa) [reply docs/conf.py]: Yw and thank you. :)

**Review by makermelissa — APPROVED** [review]

**Pull request merged**